src/genegraph/transform/gene_validity_refactor.clj

```diff
@@ -48,8 +48,7 @@
                construct-earliest-articles
                construct-secondary-contributions
                construct-variant-score
-               construct-unscoreable-evidence
-               )
+               construct-unscoreable-evidence)
```

Comment:
Not a big deal, but I like the paren on it's own separate line here to do the nature of testing in the repl by commenting out all but a few of the constructs.

Reply:
Try using `#_`.